oak_attestation_verification/src/verifier.rs

```diff
@@ -783,12 +783,11 @@ fn verify_text(
                 &public_keys.rekor_public_key,
             )?;
             // Compare the actual command line against the one inlined in the endorsement.
-            if actual.as_bytes() != &endorsement.subject {
-                anyhow::bail!("unexpected endorsement usage");
-            }
-            Ok(())
+            let regex = String::from_utf8(endorsement.subject.clone())
+                .expect("endorsement subject is not utf8");
+            verify_regex(actual, &regex)
```

Comment:
Maybe add a context to make it clear where the regex was in case the verification fails